api/citations/utils.py

```diff
@@ -222,7 +221,7 @@ def remove_extra_period_after_right_quotation(cit):
 
 def chicago_reformat(node, cit):
     cit = remove_extra_period_after_right_quotation(cit)
-    new_csl = cit.split('20')
+    new_csl = cit.split('20', 1)
```

Comment:
What was this change for?

Reply:
Ah that got hotfix https://github.com/CenterForOpenScience/osf.io/pull/8470